pkgs/development/interpreters/dart/default.nix

```diff
@@ -1,87 +1,34 @@
-{ stdenv, fetchurl, unzip, version ? "2.7.2" }:
-
-let
-
-  sources = let
-
+{ stdenv
+, fetchurl
+, unzip
+, version ? "2.10.4"
+, sources ?
```

Comment:
Adding this here so the user can point to other versions like `beta` or `dev`.

Reply:
Is it valid to do ``?`` with null or false or empty string?

Reply:
I added some basic assertions, but there is also some checks done later on: https://github.com/NixOS/nixpkgs/blob/31eb0f60571a460dda89ae2c2757a5e43bf78a50/pkgs/development/interpreters/dart/default.nix#L47